frontend/src/app/main/ui/ds/foundations/typography.stories.jsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+import Components from "@target/components";
+
+const { Heading } = Components;
+const { StoryWrapper, StoryHeader, StoryGrid, StoryGridCell } = Components.storybook;
+const { typography } = Components.meta;
+
+export default {
+  title: "Foundations/Typography",
+  component: Components.StoryHeader,
+};
+
+const typographyList = Object.entries(typography)
+  .map(([_, value]) => value)
+  .sort();
+
+export const AllTypography = {
+  render: () => (
+    <StoryWrapper theme="default">
+      <StoryHeader>
+        <h1>All Typography</h1>
+        <p>Hover on a heading to see its ID</p>
+      </StoryHeader>
+      <StoryGrid size="100%">
+        {typographyList.map((typographyId) => (
+          <StoryGridCell
+            title={typographyId}
+            key={typographyId}>
+            <Heading level="1" typography={typographyId}>
+              Lorem ipsun dolor est
+            </Heading>
+          </StoryGridCell>
+        ))}
+      </StoryGrid>
```

Comment:
This is meant for a image gallery, or when we have many small components together. 

For lines of text, we don't need this type of grid.